src/components/Calculator.js

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
```

Comment:
- [ ] **Optional:** You can use `import React, { Component } from 'react';` instead of `import { Component } from 'react';`